app/models/order_item.rb

```diff
@@ -0,0 +1,8 @@
+class OrderItem < ApplicationRecord
+  belongs_to :order
+  belongs_to :outing
+
+  def subtotal
+    self.outing.adjusted_cost * self.quantity
+  end
```

Comment:
@drod1000 I would suggest to follow the same group of methods as it has been already created for `cart` model: we had a total, which was the numeric representation and had `total_currency` where we had the currency formatted total for the views. Here I think we could have this `subtotal` for the number and `subtotal_currency` for the view. What do you think? @Dpalazzari @njgheorghita